src/test/java/core/basesyntax/LamdaCalculatorTest.java

```diff
@@ -0,0 +1,281 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class LamdaCalculatorTest {
+    private static final double DELTA = 0.0001;
+    private static final char ADDITION = '+';
+    private static final char SUBSTRACTION = '-';
+    private static final char DIVISION = '/';
+    private static final char MULTIPLIKATION = '*';
+    private static final char RAISING = '^';
+    private static LamdaCalculator lamdaCalculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        lamdaCalculator = new LamdaCalculator();
+    }
+
+    @Test
+    void calculating_sumOfTwoPositive_Ok() {
```

Comment:
```suggestion
    void calculate_sumOfTwoPositive_Ok() {
```
You should use the name of the method you are testing.